public/app/plugins/panel/heatmap/panelcfg.cue

```diff
@@ -79,7 +79,9 @@ composableKinds: PanelCfg: lineage: {
 			// Controls tooltip options
 			HeatmapTooltip: {
 				// Controls how the tooltip is shown
-				mode: ui.TooltipDisplayMode
+				mode:              ui.TooltipDisplayMode
+				maxTooltipHeight?: number
+				maxTooltipWidth?:  number
```

Comment:
can we rename these to maxHeight and maxWidth?

we're already inside tooltip options, so no need to repeat "tooltip"